portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +31,131 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Add a random fact about me to the page
+ */
+function addRandomFact() {
+    const facts = [
+        'I am a foodie',
+        'I have a collection of Starbucks mugs',
+        'I have a collection of "I LOVE" T-shirt from around the world',
+        'I have travelled to over 15 different countries'
+    ];
+
+    //Pick a random fact
+    const fact = facts[Math.floor(Math.random() * facts.length)];
+
+    //Add it to the page
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = fact;
+}
+
+isHobbiesDisplayed = false;
+isLovesDisplayed = false;
+isHatesDisplayed = false;
+
+/**
+ * Display my hobbies
+ */
+function getHobbies() {
```

Comment:
These functions seem to work well, congrats on getting them working,
something to maybe think about for a future PR is they are quite large, and they add in a lot of static html code. If you're planning on making the hobbies list dynamic or something like that ignore this suggestion, but if they're going to stay as static lists can I suggest you look into maybe adding or removing the hidden attributes from the outer tag instead? I think it would simplify this a lot. https://www.w3schools.com/tags/att_hidden.asp